research/build-reverse-dependencies/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Build reverse dependencies in Packit
+authors: mmassari
+---
```

Comment:
(self-promotion) I was starting to implement this as a tmt-plugin ([tmt-copr](https://github.com/LecrisUT/tmt-copr) and also tmt-koji, but need to finish 1 first :D)
- Still requires testing-farm integration to pull in additional tmt plugins
- Less to manage on packit side
- More control on how to display results, logs
- Needs the secret management of testing-farm to get the api token to interact with services. But an alternative could incorporate packit cli and packit service to get things working faster

This can solve some of the issues of "Build our own solution", especially w.r.t. viewing the results